*** tests/A-10-04-M-ok.out	Fri Oct 18 19:27:35 2024
--- tests/A-10-04-M-ok.outhyp	Wed Oct 23 11:36:55 2024
***************
*** 57,66 ****
  3 - Actos médicos de um veterinário
  4 - Vacinações com problemas
  0 - Sair
! Escolha uma opção: Identificador único do habitat: ANIMAL|A1|Pinguim 1|C1|VOID|AR2
! ANIMAL|A2|Pinguim 2|C1|VOID|AR2
! ANIMAL|A4|Macaco 2|C2|VOID|AR2
! Consultas
  1 - Animais de um habitat
  2 - Actos médicos num animal
  3 - Actos médicos de um veterinário
--- 57,63 ----
  3 - Actos médicos de um veterinário
  4 - Vacinações com problemas
  0 - Sair
! Escolha uma opção: Identificador único do habitat: Consultas
  1 - Animais de um habitat
  2 - Actos médicos num animal
  3 - Actos médicos de um veterinário
