*** tests_intermediate_private/07.out	Fri Oct 11 02:11:43 2024
--- tests_intermediate_private/07.outhyp	Fri Oct 11 12:56:52 2024
***************
*** 450,456 ****
  10 - Consultas
  0 - Sair
  Escolha uma opção: 3
- Ficheiro sem nome. Guardar ficheiro como: test7.dat
  Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
--- 450,455 ----
***************
*** 463,468 ****
--- 462,469 ----
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
+ Escolha uma opção: test7.dat
+ Número inválido!
  Escolha uma opção: 3
  Menu Principal
  1 - Criar ficheiro
